client/src/components/EventCard.tsx

```diff
@@ -9,26 +9,17 @@ import {
 } from '@chakra-ui/react';
 import { Link } from 'chakra-next-link';
 import React from 'react';
-import { Chapter, Event, EventTag } from '../generated/graphql';
+import { EventWithRelations } from '../generated/graphql';
 import { formatDate } from '../util/date';
 
 type EventCardProps = {
-  event: Pick<
-    Event,
-    | 'id'
-    | 'name'
-    | 'description'
-    | 'start_at'
-    | 'image_url'
-    | 'invite_only'
-    | 'canceled'
-  > & {
-    chapter: Pick<Chapter, 'id' | 'name'>;
-    tags?: EventTag[] | null;
-  };
+  event: EventWithRelations;
```

Comment:
How about this. Use 
```suggestion
 event: ChapterQuery['chapter']['events'];
```
since that's what you're getting from the chapterPage and an optional `eventUsers` prop that you only pass in when you want to show this info?  Unless the plan is to show it everywhere, in which case it's fine to update the type to include `event_users`.

Reply:
[b05355a](https://github.com/freeCodeCamp/chapter/pull/1630/commits/b05355ab5858b9ad5db7c0c803859b94df5d3573) there is a change in types that make the solution not good enough, I will come back to this after fixing emails feedback, and I want to display users profiles as well so it was going to take a while anymore 👍